docs/configuration.md

```diff
@@ -1139,13 +1148,12 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.driver.bindAddress</code></td>
   <td>(value of spark.driver.host)</td>
   <td>
-    <p>Hostname or IP address where to bind listening sockets. This config overrides the SPARK_LOCAL_IP
-    environment variable (see below).</p>
-
-    <p>It also allows a different address from the local one to be advertised to executors or external systems.
+    Hostname or IP address where to bind listening sockets. This config overrides the SPARK_LOCAL_IP
```

Comment:
This will now be rendered as a long paragraph, won't it? I kinda prefer having the break.